File: jawa/yoda/yoda-bot/src/main/java/ru/prohor/universe/yoda/bot/Text.java
package ru.prohor.universe.yoda.bot;

import org.slf4j.helpers.MessageFormatter;

/**
 * Класс-шаблонизатор текста, с помощью которого бот взаимодействует с пользователем
 */
public class Text {
    private static String format(String pattern, Object... args) {
        return MessageFormatter.arrayFormat(pattern, args).getMessage();
    }

    /**
     * Описания для команд (используется в команде /help
     * {@link ru.prohor.universe.yoda.bot.handlers.command.HelpCommandHandler HelpCommandHandler})
     */
    public static class CommandDescriptions {
        public static String start() {
            return "команда для запуска или перезапуска бота";
        }

        public static String help() {
            return "список команд и справка";
        }
    }

    /**
     * Ответы на вызовы команд ({@link ru.prohor.universe.jocasta.tgbots.api.comand.CommandHandler CommandHandler})
     */
    public static class CommandReplies {
        public static String nonCommand(String text) {
            return format("Я не знаю что делать с сообщением \"{}\"", text);
        }

        public static String start() {
            return "Приветствую! Помогать с занятиями вам буду я";
        }

        public static String unknown() {
            return "Неизвестная команда. Список доступных команд - /help";
        }
    }
}

File: jawa/yoda/yoda-bot/src/main/kotlin/Text.kt
package ru.prohor.universe.yoda.bot

import ru.prohor.universe.jocasta.tgbots.api.comand.CommandHandler

/**
 * Класс-шаблонизатор текста, с помощью которого бот взаимодействует с пользователем
 */
object Text {
    /**
     * Команды, используемые в боте
     */
    object Commands {
        const val START = "/start"
        const val HELP = "/help"
    }

    /**
     * Описания для команд (используется в команде /help
     * [HelpCommandHandler][ru.prohor.universe.yoda.bot.handlers.command.HelpCommandHandler])
     */
    object CommandDescriptions {
        fun start(): String {
            return "команда для запуска или перезапуска бота"
        }

        fun help(): String {
            return "список команд и справка"
        }
    }

    /**
     * Ответы на вызовы команд ([CommandHandler][CommandHandler])
     */
    object CommandReplies {
        fun nonCommand(text: String?): String? {
            return "Я не знаю что делать с сообщением \"$text\""
        }

        fun start(): String {
            return "Приветствую! Помогать с занятиями вам буду я"
        }

        fun unknown(): String {
            return "Неизвестная команда. Список доступных команд - /help"
        }
    }
}
